release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -3354,6 +3354,57 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+//4.16.39
+[id="ocp-4-16-39_{context}"]
+=== RHSA-2025:3301 - {product-title} {product-version}.39 bug fix and security update
+
+Issued: 20 April 2025
+
+{product-title} release {product-version}.39 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4008[RHSA-2025:4008] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4010[RHBA-2025:4010] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.16.39 --pullspecs
+----
+
+[id="ocp-4-16-39-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, containers that used the SELinux `container_logreader_t` domain for the purposes of viewing container logs on a host at `/var/log` could not access logs in the `/var/log/containers` subdirectory. This issue occurred because of a missing symbolic link. With this release, a symbolic link is created for `/var/log/containers` so that containers can access the logs in `/var/log/containers`. (link:https://issues.redhat.com/browse/OCPBUGS-54343[OCPBUGS-54343]) 
+
+* Previously, when the `openshift-install agent create pxe-files` command was run, the command created temporary directories in `/tmp/agent` and the command did not remove these directories upon completion. With this release, the command now removes the directories when the command nears completion, so manual deletion of directories is not necessary. (link:https://issues.redhat.com/browse/OCPBUGS-52961[OCPBUGS-52961])
+
+* Previously, the error message "unable to read image" displayed when building the agent ISO in a disconnected setup even though the image was created. This was due to the `ImageContentSourcePolicy` (ICSP) not being checked. With this release, the error message no longer displays. (link:https://issues.redhat.com/browse/OCPBUGS-53378[OCPBUGS-53378])
+
+* Previously, the cluster autoscaler stopped scaling because a machine failed in a machine set. This situation happened because of inaccuracies in the way the cluster autoscaler counts machines in various non-running phases. With this release, the inaccuracies have been fixed so that the cluster autoscaler has a more accurate count. (link:https://issues.redhat.com/browse/OCPBUGS-54325[OCPBUGS-54325]) 
```

Comment:
```suggestion
* Previously, the cluster autoscaler stopped scaling because a machine failed in a machine set. This situation happened because of inaccuracies in the way the cluster autoscaler counts machines in various non-running phases. With this release, the inaccuracies have been fixed so that the cluster autoscaler has a more accurate count. (link:https://issues.redhat.com/browse/OCPBUGS-54326[OCPBUGS-54326]) 
```

The Jira needed the following fields to be updated:

![Screenshot From 2025-04-22 09-43-32](https://github.com/user-attachments/assets/10e588b1-9373-4afc-8c37-3adb85da080e)